Doberman/ControlNode.py

```diff
@@ -84,6 +86,7 @@ def control_pipeline(self, action, pipeline):
             target = 'pl_convert'
         else:
             raise ValueError(f'Don\'t know what to do with pipeline {pipeline}')
+        self.logger.debug(f"Sending {action} to {pipeline}")
         self.pipeline.send_command(command=f'pipelinectl_{action} {pipeline}',
-                to=target, issuer=self.pipeline.name)
```

Comment:
Why no issuer?

Reply:
send_command doesn't accept it:

https://github.com/AG-Schumann/Doberman/blob/fb0bca14be49db0f2bbe0c919b0778cb3af6c36d/Doberman/Pipeline.py#L286-L293